person.rb

```diff
@@ -1,6 +1,7 @@
 require './nameable'
 require './capitalize_decorator'
 require './trimmer_decorator'
+require './rental'
 
 class Person < Nameable
   attr_reader :id
```

Comment:
- [ ] Kindly make sure to set up the `rentals` getters and setters, you can do this by adding the `rentals` to the `attr_accessor`

Reply:
Thanks for the review @Ismailco. I'll do all the changes 😁